.travis.yml

```diff
@@ -1,15 +1,51 @@
 language: cpp
 
-env:
+#env:
   #global:
   #  # COVERITY_SCAN_TOKEN
   #  - secure: "HK0Ey4+9OsxcWl0FwNzwbkEkvHMD7DM27x3NWeY8Lo2NmU2ZBB72Q3TWIBtk6YpeY5glsBTUQob17H+ISQR2xHqL5GaGf1hWk9x7iBAA4XRb4RUgCnzEE6vPmUGcxcjx0AIjtbnFKShoEOKbN/O2ms1zqz8rgUTNiAQWFgJKCW0="
-  - tbs_arch=x86
-  - tbs_arch=x64
 
-os:
-  - linux
-  - osx
+matrix:
```

Comment:
this matrix is way over the top imo.  there's no need to split up the sanitizer logic this much.  just do a normal build and a debug build, and for the debug build, include all the sanitizers at once.